src/pad.ts

```diff
@@ -676,12 +687,14 @@ UI.pad.notepad = function (dom, padDoc, subject, me, options) {
     return part
   }
 
-  var newChunk = function (ele, before) {
+  /* @@ TODO we need to look at indent, it can be a Number or an Object this doesn't seem correct.
```

Comment:
Any advice on indent or should I just leave as is?  I had to change a bit here too because indent could either be a number or an object.

Reply:
Definitely something you should fix as part of the cleanup. In which cases is it a number and in which cases is it an object? When it's an object, then what does that object contain? Can you convert it to a number?

Reply:
I see what you are saying. I will see if I can determine this as I write my examples.

Reply:
Added a comment for now in the code.